source/slang/glsl.meta.slang

```diff
@@ -7263,3 +7263,700 @@ __glsl_version(430) [require(glsl)]
     }
     }
 }
+
+/// Section 8.14. Fragment Processing Functions
+
+__glsl_version(430)
+__spirv_version(1.0)
+[require(any_gfx_target, fragmentprocessing)]
```

Comment:
Can this be just `[require(any_gfx_target, fragment)]`

Reply:
I would assume not since we need a way to enforce glsl/hlsl versioning requirements?